modules/@apostrophecms/settings/index.js

```diff
@@ -1,12 +1,53 @@
+const { klona } = require('klona');
+
 module.exports = {
   options: {
     alias: 'settings',
     subforms: {},
-    // XXX not implemented until Phase 2
     groups: {}
   },
 
   init(self) {
+    // List of all allowed protected types and their aliases (`subform.protected: type`).
+    // The key is the type or alias, the value is the actual type (always a string).
+    // All subforms `protected` prop will be converted to the actual type.
+    // Invalid protection types will panic.
+    self.protectedTypes = {
+      true: 'password',
```

Comment:
This line specifically needs a comment. "Protection type to be used if `protected` is simply set to `true`."

Reply:
Let's change the `protected` property to `protection` because it is more than just a boolean now, it specifies a specific kind of protection, although `true` can be used to mean "what the system thinks is best for most fields." That will help with the confusion I experienced in the earlier code above.

Reply:
I can't agree more. I'll provide a patch now.